### DISCUSSION TITLE
Model never loads in AR on iOS QuickLook.

### DISCUSSION THREAD

**Discussion by Shubhra22** [google/model-viewer#4506] [Q&A]
### Description
QuickLook keeps loading forever for glb models. 
![IMG_3393](https://github.com/google/model-viewer/assets/2692591/38a8b1f6-3776-4251-bedc-31365e789e85)

Works fine on Android. I tried to convert it manually to usdz and then it works. 

#### Live Demo

https://polymuse.onrender.com/ar/?id=b0NMq6uu5Cw1trAoVsTg

### Version

- model-viewer: v3.2.1

### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [x] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [ ] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [x] QuickLook

**Comment by Shubhra22** [comment]
Update: Interestingly, I tried another glb file and that works. But the model I am using is not working. 

My model is a scanned object from a 3d scanning app from iOS. I compressed the file (https://github.khronos.org/glTF-Compressor-Release/) with ktx2 texture. But still that does not help.

**Reply by elalish** [reply]
The forever load sounds like a QuickLook bug (@grorg), but in general we don't do a perfect conversion of all GLBs to USDZ, it's only best effort. If it fails, please use Apple's converters and specify your own USDZ as the `ios-src`.

**Reply by Shubhra22** [reply]
Thanks for the reply. Can you provide me some directions on the conversion? I honestly don’t have much clue where to start. I am using Unity and I found a few editor solution, but didn’t find any runtime converters. It seems like I have to write my own.

**Reply by elalish** [reply]
Best bet is find an Apple machine and use their free Reality Converter software.

**Reply by Shubhra22** [reply]
I tried it on other converter and it works fine. Seems like the usdz converter that comes with modelviewer may have some issue.
